sql_generators/glean_usage/__init__.py

```diff
@@ -34,11 +35,74 @@
     clients_last_seen_joined.ClientsLastSeenJoined(),
 ]
 
-# * mlhackweek_search was an experiment that we don't want to generate tables
-# for
-# * regrets_reporter currently refers to two applications, skip the glean
+# regrets_reporter currently refers to two applications, skip the glean
 # one to avoid confusion: https://github.com/mozilla/bigquery-etl/issues/2499
-SKIP_APPS = ["mlhackweek_search", "regrets_reporter", "regrets_reporter_ucs"]
+SKIP_APPS = ["regrets_reporter", "regrets_reporter_ucs"]
```

Comment:
`mlhackweek_search` is now deprecated, so we don't need to explicitly skip it anymore.